sceptre/stack.py

```diff
@@ -116,6 +116,10 @@ class Stack(object):
     :param stack_group_config: The StackGroup config for the Stack
     :type stack_group_config: dict
 
+    :param duration_seconds: The session duration when Scetre assumes a role.\
+           If not supplied, Sceptre uses default value (3600 seconds)
+    :type duration_seconds: int
```

Comment:
It's best to just use a type annotation to indicate types rather than explicitly documenting them in the docstrings.